anomalib/utils/callbacks/min_max_normalization.py

```diff
@@ -12,12 +12,18 @@
 
 from anomalib.models.components import AnomalyModule
 from anomalib.post_processing.normalization.min_max import normalize
+from anomalib.utils.metrics import MinMax
 
 
 @CALLBACK_REGISTRY
 class MinMaxNormalizationCallback(Callback):
     """Callback that normalizes the image-level and pixel-level anomaly scores using min-max normalization."""
 
+    def setup(self, _trainer: pl.Trainer, pl_module: AnomalyModule, _stage: Optional[str] = None) -> None:
+        """Adds min_max metrics to normalization metrics."""
+        if not hasattr(pl_module, "normalization_metrics"):
```

Comment:
Should it throw an error if the attribute already exists? Otherwise it would add the collected outputs to the existing metric in the `on_test_start` step which would mess up the computation of the other normalization metric.